### ISSUE TITLE
win32 jenkins build broken

### ISSUE THREAD

**Issue by Aljutor** [qTox/qTox#298]
win32 jenkins build broken:
https://jenkins.libtoxcore.so/job/qTox-win32/ 
"Last successful build (#316), 19  days ago"

**Comment by tux3** [comment]
Unfortunately I'm not in charge of Jenkins, I don't even have access to it, sorry. You should go shout at NikolaiToryzin (stqism) on #tox and #tox-dev at irc.freenode.net

That said, I'll probably eventually release my own Linux and Windows builds on GitHub, since it's becoming painfully clear that Jenkins will never be fixed.

**Comment by Aljutor** [comment]
Thank you, but I tried IRC sometime ago and I found the channels empty and abandoned.
Bad that Github has removed private messages.

**Comment by dubslow** [comment]
> Thank you, but I tried IRC sometime ago and I found the channels empty and abandoned.

That is sometimes the case, especially when Europe and Americas are asleep, but there's usually activity at least once an hour for most hours of the day. Between 1200-0600 UTC are the most likely hours (Europeans in the earlier part, Americans in the latter), but NikolaiToryzin (and me, for that matter) have been known to be around during the awkward hours of 0600-1200 UTC.

tl;dr you had some bad luck, irc is usually pretty active (also you wouldn't be the first to yell at stqism and probably not the last)
